core/block_dragger.js

```diff
@@ -227,3 +231,29 @@ Blockly.BlockDragger.prototype.updateCursorDuringBlockDrag_ = function() {
     }
   }
 };
+
+/**
+ * TODO (fenichel): Consider putting this in utils.
+ * Convert a coordinate object from pixels to workspace units, including a
+ * correction for mutator workspaces.
+ * This function does not consider differing origins.  It simply scales the
+ * input's x and y values.
+ * @param {!goog.math.Coordinate} pixelCoord A coordinate with x and y values
+ *     in css pixel units.
+ * @return {!goog.math.Coordinate} The input coordinate divided by the workspace
+ *     scale.
+ * @private
+ */
+Blockly.BlockDragger.prototype.pixelsToWorkspaceUnits_ = function(pixelCoord) {
+  var result = new goog.math.Coordinate(pixelCoord.x / this.workspace_.scale,
+      pixelCoord.y / this.workspace_.scale);
+  if (this.workspace_.isMutator) {
+    // If we're in a mutator, its scale is always 1, purely because of some
```

Comment:
This comment makes it sound like we did this on purpose to make things render faster. I don't think that's the case. I think it just happened to work when it was originally written. If I had infinite time, I'd probably try and clean this up so coordinates weren't so confusing in mutators.